scripts/Send-Signal.ps1

```diff
@@ -0,0 +1,68 @@
+#######################################################################################################################
+# Send-Signal.ps1
+# 
+# Parse System event log for specifc log and source. Can be configured to run on a schedule via Task Scheduler, but
+# this reuires Python27 to be in the PATH environment variable.
+#
+# Examples:
+#     PS> Send-Signal "System" "ExtMirr" 5
+#     PS> Send-Signal -EventLog "System" -EventSource "ExtMirr" -MinutesPrevious 5
+#
+#######################################################################################################################
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$True, Position=0)]
+    [string] $EventLog = "System",
+
+    [Parameter(Mandatory=$True, Position=1)]
+    [string] $EventSource = "ExtMirr",
+
+    [Parameter(Mandatory=$False, Position=2)]
+    [int] $MinutesPrevious = 5
+)
+
+$pyscript = ".\report_event.py"
+$node = $env:COMPUTERNAME
+
+$xMinutesAgo = (Get-Date).Subtract((New-TimeSpan -Minutes $MinutesPrevious))
+
+$events = $null

```

Comment:
Why is this null needed?

Reply:
It's not really, I just like being explicit. The next line (Get-EventLog) should return null if it doesn't find anything. I'll delete it since it offends you.

Reply:
very c++ like. I like it! 👍